docs/integrations/sources/mysql.md

```diff
@@ -183,52 +183,53 @@ If you do not see a type in this list, assume that it is coerced into a string.
 
 ## Changelog
 
-| Version | Date       | Pull Request                                               | Subject                                                                                                          |
```

Comment:
Please try to avoid changing all change log when you add just 1 line, it's can mask some not obvious changes

Reply:
thanks for noticing, I'll rollback it now